js-challenge/bbs-api/register.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const p = console.log;
+const host = 'http://localhost'; //ローカルで掲示板を動かす場合の設定
+
+//以下にコードを書きましょう。
+const fetchForm = document.getElementById("fetchRegister");
+const btn = document.querySelector('.btn');
+
+const postRegister = () => {
+    const formData = new FormData(fetchForm);
+    fetch("http://localhost/register",
+        {
+            method: "POST",
+            body: formData
+        })
+        .then(response => {
+            return response.json();
+        })
+        .then(data => {
+            console.log(data);
+        })
+        .catch(error => {
+            console.log("失敗しました")
+        })
```

Comment:
```suggestion
        });
```
セミコロンが無いですね。セミコロンも自動整形で保管しましょう。